js/controllers/hotkeys.js

```diff
@@ -104,14 +103,3 @@ HotkeysController.prototype.onKeydown_ = function(e) {
     }
   }
 };
-
-HotkeysController.prototype.onMouseWheel_ = function(e) {
```

Comment:
why is this deleted?

Reply:
https://bugs.chromium.org/p/chromium/issues/detail?id=1007457&q=owner%3Ame&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified

Can move this to a seperate PR if you like!

Reply:
Will move to a seperate PR.